enterprise/frontend/src/embedding-sdk/lib/plugins/dashboard.ts

```diff
@@ -0,0 +1,36 @@
+import { merge } from "lodash";
+
+import type {
+  SdkDashcardMenuPluginsConfig,
+  SdkPluginsConfig,
+} from "embedding-sdk";
+
+const DEFAULT_DASHCARD_MENU_ITEMS: SdkDashcardMenuPluginsConfig = {
+  dashcardMenu: {
+    withDownloads: true,
+    withEditLink: true,
+    customItems: [],
+  },
+} as const;
+
+const getDashcardMenu = (plugins: SdkPluginsConfig) => {
+  if (typeof plugins?.dashboard?.dashcardMenu === "function") {
+    return plugins?.dashboard?.dashcardMenu;
+  }
+  return merge(
+    {},
+    DEFAULT_DASHCARD_MENU_ITEMS,
+    plugins?.dashboard?.dashcardMenu,
+  );
+};
+
+export const initializeDashboardPlugin = (
```

Comment:
nit: _initialize_ has a notion of it's doing some work before the thing would be ready. If we compare this to initializing query builder.
https://github.com/metabase/metabase/blob/3c95de91cc652ab417a65ad49739aa1661df123c/frontend/src/metabase/query_builder/actions/core/initializeQB.ts#L244-L254

you'll see that we perform a bunch of actions there, but not here.

I'd rather name this something more boring and straightforward like.

```suggestion
export const addDefaultDashboardPluginValues = (
```